src/js/osweb/items/sketchpad.js

```diff
@@ -40,6 +40,11 @@ export default class Sketchpad extends GenericResponse {
     this.elements = []
     this.vars.duration = 'keypress'
   }
+  
+  /** Process a time out response. */
+  process_response_timeout () {
+    // Nothing happens

```

Comment:
Seems quite odd to add an empty function? I guess this is a function that is called for any item at the end of some period, but maybe it is better then to move this function to the parent (Item) ?

Reply:
This is necessary to override this function from `generic_response`, which modifies the response variables (which we don't want on a timeout for a `sketchpad`, because that means that the `sketchpad` was set to a numeric duration).